src/_pytest/pytester.py

```diff
@@ -665,6 +666,17 @@ def maketxtfile(self, *args, **kwargs):
         """Shortcut for .makefile() with a .txt extension."""
         return self._makefile(".txt", args, kwargs)
 
+    def makefiles(self, **kwargs: str) -> Optional[Path]:
+        """Create the given set of files."""
+        first_path = None
+        for fname, content in kwargs.items():
+            path = Path(self.tmpdir).joinpath(Path(fname))
+            with open(path, "w") as fp:
+                fp.write(textwrap.dedent(content))
+            if not first_path:
+                first_path = path
+        return first_path
```

Comment:
I think it might be better to actually return either all created paths, or None always maybe.